app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -171,15 +171,30 @@ public boolean onInterceptTouchEvent(@NonNull RecyclerView rv, @NonNull MotionEv
         return false;
       }
 
+      /**
+       * Process a touch event as part of a gesture that was claimed by returning true
+       * from a previous call to {@link #onInterceptTouchEvent}.
+       *
+       * @param rv
+       * @param e  MotionEvent describing the touch event. All coordinates are in
```

Comment:
Missing the end of the sentence I guess? :-)

Reply:
done.